### DISCUSSION TITLE
Github Pages server always specifies utf-8 in header regardless of meta tag

### DISCUSSION THREAD

**Discussion by kot-ch** [community/community#22867] [Other Feature Feedback, Questions, & Ideas]
<p>Hi,</p>
<p>I’ve written a small Web page: <a href="http://flaz14.github.io/downgrade/index.html" rel="nofollow noopener">http://flaz14.github.io/downgrade/index.html</a></p>
<p>That page is for Downgrade topics. So I would like to use <strong>cp866</strong> as page encoding (because it has been used in DOS as encoding for Russian language in good old times). I’ve specified the encoding inside <strong>meta</strong> tag like this (BTW I’m using HTML 4.01 Strict):</p>
<pre><code>\&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd"\&gt;\&lt;html\&gt;\&lt;head\&gt;\&lt;meta http-equiv="Content-Type" content="text/html; charset=cp866"\&gt;...\&lt;/head\&gt;  
...
</code></pre>
<p>When I load the page in a Web-browser (I’ve tried <em>Firefox</em> and <em>Lynx</em>) from Github Pages server I cannot read Russian  letters: they are bloated (so, for example, in Firefox I need to go to menus and choose <em>Cyrillic (DOS)</em> by hand). But when I load the page from my local disk everything looks fine.</p>
<p>I’ve investigated this and found out that Github Pages server supplies <strong>text/html; charset=utf-8</strong> in <strong>Content-Type</strong> header regardless of the charset specified in <strong>meta</strong> tag.</p>
<p>Can I use <strong>cp866</strong> as page encoding at Github Pages?        </p>
<p>PS I also tried another synonym but without luck:</p>
<pre><code>&lt;meta http-equiv="Content-Type" content="text/html; charset=DOS-866"&gt;
</code></pre>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/flaz14">@flaz14</a>,</p>
<p>Thanks for being part of the GitHub Community Forum.</p>
<p>To answer your question, we only support utf-8 at this time. We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ll share your request for cp866 encoding support with the appropriate team, though I can’t guarantee anything.</p>
<p>Cheers!</p>

**Comment by ivan386** [comment]
<p>I am not found way change default encoding from utf-8 to koi-8r.</p>
